### ISSUE TITLE
[Bug] Backdrop not fading away while dragging the sheet

### ISSUE THREAD

**Issue by naamapps** [fujidaiti/smooth_sheets#78]
An issue I encountered is when dragging the sheet, the backdrop fades away ever so slightly instead of fully fading away.
Strangely though, is that when dragging down, and then up all the way to the top of the screen, and then down again to the bottom, the backdrop fades completely fine.

See here:

https://github.com/fujidaiti/smooth_sheets/assets/37176893/ca3a5c61-a6ba-4936-b871-22dfe3df6f39

**Comment by fujidaiti** [comment]
It works fine except that the color changes are too subtle to notice. Try again with a larger sheet and then you will see the barrier fades out more clearly. ( I know it needs more improvement, though...)

**Comment by naamapps** [comment]
Yes you're right, however, I would probably expect that the fading effect would be based of the sheet height rather than the device screen height.
Not critical really, just something I noticed :)
